tests/src/CompareEDM4hepLCIO.cc

```diff
@@ -33,35 +49,44 @@ bool compare(const EVENT::CalorimeterHit* lcioElem, const edm4hep::CalorimeterHi
   return true;
 }
 
-bool compare(const lcio::LCCollection* lcioCollection, const edm4hep::CalorimeterHitCollection& edm4hepCollection)
+bool compare(
+  const lcio::LCCollection* lcioCollection,
+  const edm4hep::CalorimeterHitCollection& edm4hepCollection,
+  const ObjectMappings& objectMaps)
 {
-  return compareCollection<EVENT::CalorimeterHit>(lcioCollection, edm4hepCollection);
+  return compareCollection<EVENT::CalorimeterHit>(lcioCollection, edm4hepCollection, objectMaps);
 }
 
 // ================= Cluster ================
 
-bool compare(const EVENT::Cluster* lcioElem, const edm4hep::Cluster& edm4hepElem)
+bool compare(const EVENT::Cluster* lcioElem, const edm4hep::Cluster& edm4hepElem, const ObjectMappings& objectMaps)
 {
   ASSERT_COMPARE(lcioElem, edm4hepElem, getType, "type in Cluster");
   ASSERT_COMPARE(lcioElem, edm4hepElem, getEnergy, "energy in Cluster");
   ASSERT_COMPARE(lcioElem, edm4hepElem, getEnergyError, "energyError in Cluster");
   ASSERT_COMPARE(lcioElem, edm4hepElem, getPosition, "position in Cluster");
   ASSERT_COMPARE(lcioElem, edm4hepElem, getPositionError, "positionError in Cluster");
   ASSERT_COMPARE(lcioElem, edm4hepElem, getITheta, "iTheta in Cluster");
-  // TODO: LCIO has getIPhi not get Phi
-  // ASSERT_COMPARE(lcioElem, edm4hepElem, getPhi, "phi in Cluster");
+  // LCIO has getIPhi and EDM4hep has getPhi
+  ASSERT_COMPARE_VALS(lcioElem->getIPhi(), edm4hepElem.getPhi(), "phi in Cluster");
```

Comment:
Maybe for the future we should change the comparison in `ASSERT_COMPARE_VALS` as comparing for equality floating point numbers doesn't work always

Reply:
I agree in principle, but in this case since we are checking a conversion I would like to have floating point equality down to the bit and not just "close enough".

Reply:
I guess with LCIO being quite similar we didn't hit problems as converting from double -> float as this wouldn't work with that comparison